pkg/operator/observe_config.go

```diff
@@ -24,11 +25,18 @@ import (
 	"k8s.io/client-go/util/flowcontrol"
 	"k8s.io/client-go/util/workqueue"
 
+	"github.com/openshift/api/operator/v1alpha1"
 	operatorconfigclientv1alpha1 "github.com/openshift/cluster-kube-apiserver-operator/pkg/generated/clientset/versioned/typed/kubeapiserver/v1alpha1"
 	kubeapiserveroperatorinformers "github.com/openshift/cluster-kube-apiserver-operator/pkg/generated/informers/externalversions"
+	"github.com/openshift/library-go/pkg/operator/v1alpha1helpers"
 )
 
-type observeConfigFunc func(kubernetes.Interface, *rest.Config, map[string]interface{}) (map[string]interface{}, error)
+const (
+	configObservationError   = "ConfigObservationError"
+	configObservationSuccess = "ConfigObservationSuccess"
```

Comment:
no.  this will result in fighting control loops.

Reply:
Help me understand. Are we saying a single reason, such as 'ConfigObservation' ?